### PR TITLE
Johanna Rexin - Week 12 - Project Movie

### PR DESCRIPTION
**Pull request by johannarexin**

### CODE REVIEW

File: code/src/pages/MovieDetails.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <div className="movie-details-container">
+            <img className="movie-details-poster" src={`https://image.tmdb.org/t/p/original${movieDetails.poster_path}`} alt={movieDetails.title}/>
+            <div className="movie-details-information">
+              <h1 className="title">{movieDetails.title}
+              <span className="vote">{movieDetails.vote_average}/10</span></h1>
+              <p className="movie-details-overview">{movieDetails.overview}</p>
+            </div>
+          </div>
+        </div>
+      </article>
```
Comment (jonnaru) [inline code/src/pages/MovieDetails.js:37]: If you want to put the button back to movies on this page (the one with the small arrow) you can use onClick with history.goBack. Here is an example, if you scroll down a bit to the middle of the page https://reacttraining.com/blog/react-router-v5-1/

File: code/src/pages/MovieList.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//             key={movie.id} 
+//             name={movie.name} 
+//             background_image={movie.background_image}
+//             rating={movie.rating}
+//             />
+//         ))}
+//         </section>
+//     )
+// }
+
```
Comment (jonnaru) [inline code/src/pages/MovieList.js:55]: Are you going to use this for something later on or keeping it to remember something in the code? If so, maybe write a comment explaining why. Or if you're finished with it maybe just delete?

File: code/src/pages/MovieDetails.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  font-size: 20px;
+}
+
+.movie-details-overview {
+  margin-left: 20px;
+  margin-bottom: 20px;
+  color: white;
+  max-width: 400px;
+}
+
```
Comment (jonnaru) [inline code/src/pages/MovieDetails.css:43]: Don't forget the media queries for this page!

**Review by jonnaru — COMMENTED** [review]
Really good work, Johanna! Be proud :))) ⭐️🌟 It would be cool if you added that back-button in MovieDetails.js and just make sure the same page looks pretty in mobile 🤩

File: code/src/pages/MovieDetails.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      })
+    }
+    
+    useEffect (()=> {
+      getMovieDetails(id)
+    },[id]);  
+    
+    
+    return (
+      <article className="movie-details">
```
Comment (EmOlsz) [inline code/src/pages/MovieDetails.js:25]: Great class naming, congrats :)

File: code/src/pages/MovieList.js (hunk 2 above)
Comment (EmOlsz) [reply code/src/pages/MovieList.js:55]: 👍

**Pull request closed without merging**